framework/src/st_things/st_things.c

```diff
@@ -387,6 +387,42 @@ int st_things_start(void)
 	return ST_THINGS_ERROR_OPERATION_FAILED;
 }
 
+int st_things_push_notification_to_cloud()
+{
+	THINGS_LOG_D(TAG, THINGS_FUNC_ENTRY);
+	int push_ret = 0;
+	char targetUri[32 * 2] = { 0, };
+	OCRepPayload *payload = NULL;
+	OCRepPayload *data_Payload = NULL;
+	snprintf(targetUri, 32 * 2 -1, "%s", "/service/pushservice");
+	payload = OCRepPayloadCreate();
+	if (!payload) { 
+		THINGS_LOG_E(TAG, "Failed to allocate Payload");
+		return 0;
+	} else {
+		data_Payload = OCRepPayloadCreate();
+		if (!data_Payload) { 
+			THINGS_LOG_E(TAG, "Failed to allocate data_Payload"); 
+			OCRepPayloadDestroy(payload);
+			return 0;
+		} else { 
+			OCRepPayloadSetUri(payload, targetUri);
+			OCRepPayloadSetPropInt(payload, "x.org.iotivity.ns.messageid", 0);
+			OCRepPayloadSetPropString(payload, "x.org.iotivity.ns.providerid", OCGetServerInstanceIDString());
+			OCRepPayloadSetPropString(payload, "x.org.iotivity.ns.source", "/alarms/vs/0");
+			OCRepPayloadSetPropString(payload, "x.org.iotivity.ns.type", "4");
+			OCRepPayloadSetPropString(payload, "x.com.samsung.code", "DA_WF_T_0008");
+			OCRepPayloadSetPropString(payload, "x.com.samsung.code", "Light 50 % progress");
+			OCRepPayloadSetPropString(data_Payload, "et", "device.changed");
+			OCRepPayloadSetPropObject(payload, "x.com.samsung.data", data_Payload);
```

Comment:
Every properties values should be able to set by user. 
Could you please add any structure or parameter of function.